### PR TITLE
New notifications

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Includes the necessary changes to be compatible with the existing notifications on web

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: tests/test_trade_pipeline.py [hunk 1]
```diff
@@ -1163,6 +1163,60 @@ def test_order_expires_after_only_maker_messaged(self):
...
+    #     )
+    #     taker_headers = trade.get_robot_auth(trade.taker_index)
+    #     response = self.client.get(reverse("notifications"), **taker_headers)
+    #     self.assertResponse(response)
+    #     notifications_data = list(response.json())
+    #     self.assertEqual(notifications_data[0]["order_id"], trade.order_id)
+    #     self.assertEqual(
+    #         notifications_data[0]["title"],
+    #         f"⚖️ Hey {data['taker_nick']}, you lost the dispute on your order with ID {str(trade.order_id)}."
+    #     )
```
Comment (KoalaSat) [inline tests/test_trade_pipeline.py:1195]: Since the dispute resolution is an admin action, I'm not quite sure how to do it, ideas @Reckless-Satoshi ?

Comment (Reckless-Satoshi) [reply tests/test_trade_pipeline.py:1195]: I wonder whether we could directly call https://github.com/RoboSats/robosats/blob/dfd1cf53602bdcee14727661f27eea9bbd1f0a73/api/admin.py#L185 ... Looks difficult. Maybe we should explore how the request sent from the admin panel looks when solving a dispute and try to imitate it here?

Comment (KoalaSat) [reply tests/test_trade_pipeline.py:1195]: And fake the admin request with credentials? I'll take a look but for this kind of solutions they are always tricky

**Comment by KoalaSat** [comment]
Merging this changes as agreed with @Reckless-Satoshi to figure out in other PR what to do with admin actions on tests

**Comment by KoalaSat** [comment]
I don´t understand why ruff is complaining on lines doesn´t even exist anymore :upside_down_face:

**Pull request merged**